DEA_datasets/DEA_Coastlines.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
```

Comment:
<p>GIS -&gt; GIS program?</p><p> </p><p>I have never heard the noun "transect" before, so maybe consider a more well-known term here?</p><p> </p><p>There is a 50 km limitation in how long the transect can be - you should say this in the cell</p><p> </p><p>This widget is super cool, but I can't get any outputs that aren't errors! Something is unclear here :(</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/698/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>